src/Sulu/Bundle/MediaBundle/Resources/js/package.json

```diff
@@ -6,7 +6,9 @@
     "dependencies": {
         "classnames": "^2.2.5",
         "copy-to-clipboard": "^3.0.8",
+        "debounce": "^1.0.2",
```

Comment:
This can probably be removed now, right?

Reply:
No, there are other components using `debounce` as well, e.g. `ImageRenderer`